### PR TITLE
InputMapPoint/InputMapFindPlace: add parameter to hide/show geocoding…

### PR DESCRIPTION
**Pull request by kaligrafy**
… button

### CODE REVIEW

File: packages/evolution-common/src/services/widgets/WidgetConfig.ts [hunk 1]
```diff
@@ -284,6 +284,7 @@ export type InputMapPointType<CustomSurvey, CustomHousehold, CustomHome, CustomP
     CustomPerson
 > & {
     inputType: 'mapPoint';
+    showSearchPlaceButton?: boolean | ParsingFunction<string, CustomSurvey, CustomHousehold, CustomHome, CustomPerson>;
```
Comment (tahini) [inline packages/evolution-common/src/services/widgets/WidgetConfig.ts:287]: It should be `ParsingFunction<boolean>`, not string

File: packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ export type InputMapFindPlaceProps<CustomSurvey, CustomHousehold, CustomHome, Cu
     widgetConfig: InputMapFindPlaceType<CustomSurvey, CustomHousehold, CustomHome, CustomPerson>;
 };
 
-interface InputMapFindPlaceState {
+interface InputMapFindPlaceState<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> {
```
Comment (tahini) [inline packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx:41]: Does this interface need to be generic? The generic types don't appear to be used

File: packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx [hunk 3]
```diff
@@ -409,7 +409,7 @@ export class InputMapFindPlace<CustomSurvey, CustomHousehold, CustomHome, Custom
                     readOnly={true}
                     ref={this.props.inputRef}
                 />
-                {this.props.widgetConfig.refreshGeocodingLabel && (
+                {this.props.widgetConfig.refreshGeocodingLabel && this.props.widgetConfig.showSearchPlaceButton !== false && (
```
Comment (tahini) [inline packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx:412]: showSearchPlaceButton can be a ParsingFunction, I'd expect to see a call to `surveyHelper.parseBoolean` somewhere

**Review by tahini — CHANGES_REQUESTED** [review]

File: packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx (hunk 2 above)
Comment (tahini) [reply packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx:41]: > Does this interface need to be generic? The generic types don't appear to be used

The question remains here

**Review by tahini — APPROVED** [review]
Ok, it's just the matter of the generic state type, but otherwise, looks good

**Comment by kaligrafy** [comment]
Il m'a forcé à ajouter les generic types au state, il voulait pas compiler sans ça... Je ne sais pas à quoi ça sert...

**Pull request merged**